src/main/webapp/style.css

```diff
@@ -0,0 +1,7 @@
+h1 {
+    color: blue;
+    font-size: 50px;
+    font-weight: bold;
+    text-align: center;
+    padding-top: 7vh;
+}
```

Comment:
Same here,
we can have:
step1.css
step2.css
step3.css

Reply:
Share style.css across all your pages.